.github/workflows/pr.format-all.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: PR Gate - Format All
```

Comment:
This job validates formatting of all files matching the same glob pattern used in the pre-commit hook. Maybe a little aggressive, definitely verbose. 

Options to improve by adding a .prettierrc and/or .prettierignore to make it more readable for the mere mortals. And that would allow us to simplify the prettier command to just... `prettier --check .`